pkgs/tools/networking/ddclient/default.nix

```diff
@@ -1,24 +1,25 @@
-{buildPerlPackage, fetchurl, perlPackages, iproute}:
+{ stdenv, buildPerlPackage, fetchurl, perlPackages, iproute }:
 
-buildPerlPackage {
-  name = "ddclient-3.8.2";
+buildPerlPackage rec {
+  name = "ddclient-${version}";
+  version = "3.8.3";
 
   src = fetchurl {
-    url = mirror://sourceforge/ddclient/ddclient-3.8.2.tar.gz ;
-    sha256 = "17mcdqxcwa6c05m8xhxi4r37j4qvbp3wgbpvzqgmrmgwava5wcrw";
+    url = "mirror://sourceforge/ddclient/${name}.tar.gz";
+    sha256 = "1j8zdn7fy7i0bjk3jf0hxnbnshc2yf054vxq64imxdpfd7n5zgfy";
   };
 
   buildInputs = [ perlPackages.IOSocketSSL perlPackages.DigestSHA1 ];
 
-  patches = [ ./ddclient-foreground.patch ./ddclient-line-buffer-stdout.patch ];
```

Comment:
Doh! Thanks for the eagle eyes! I've force-pushed a new commit with the file removed.